### ISSUE TITLE
Problem with access

### ISSUE THREAD

**Issue by otlet** [Webbeh/WorldGuard-Events#1]
Welcome,

At the outset I would like to point out that I'm just starting my adventure with Java, so a simple topic may seem, but Google's Uncle didn't help.

I add in pom.xml:
```XML
        <dependency>
            <groupId>net.raidstone</groupId>
            <artifactId>WorldGuardEvents</artifactId>
            <version>1.14.4-R0</version>
            <scope>provided</scope>
        </dependency>
```
and i create new, simple listener:
```java
package com.idkcloud.JestemGraczem.PrivateCuboid;

import com.idkcloud.JestemGraczem.JestemGraczem;
import net.raidstone.wgevents.events.RegionEnteredEvent;
import net.raidstone.wgevents.events.RegionsLeftEvent;
import org.bukkit.entity.Player;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;

public class PrivateCuboidListener implements Listener {
    private JestemGraczem main = JestemGraczem.getPlugin(JestemGraczem.class);

    @EventHandler
    public void onRegionEntered(RegionEnteredEvent event) {
        Player p = event.getPlayer();
        if (p == null) {
            return;
        }

        if (event.getRegion().getOwners().contains(event.getUUID()) && !main.perms.has(p, "essentials.fly")) {
            main.perms.playerAdd(p, "essentials.fly");
        }
    }

    @EventHandler
    public void onRegionsLeft(RegionsLeftEvent event) {
        Player p = event.getPlayer();
        if (p == null) {
            return;
        }

        p.setFlying(false);
        if (main.perms.has(p, "essentials.fly"))
            main.perms.playerRemove(p, "essentials.fly");
    }
}
```

When i try to `mvn package` i got error:
```
Error:(4,37) java: cannot access net.raidstone.wgevents.events.RegionEnteredEvent
```

I try to `mvn install`, `mvn clean` etc. Its my first bug this type, other problems i fix.

Do any of you know how to correct this error?

**Comment by Webbeh** [comment]
Will have a look later today or tomorrow.

**Comment by Webbeh** [comment]
So, thank you for not pasting the complete error.

Since was most likely due the maven artifact being compiled in Java 10, while your project was lower than that.

It is now fixed with version 1.15.1.
